dags/data_retention_delete_dag.py

```diff
@@ -0,0 +1,87 @@
+"""
```

Comment:
I was trying to explicitly move the old dag rather than deleting (via `git mv`). `git blame` still shows the proper history of the file, but `git log` doesn't. Not sure why...